website/docs/language/state/locking.mdx

```diff
@@ -14,7 +14,7 @@ others from acquiring the lock and potentially corrupting your state.
 State locking happens automatically on all operations that could write
 state. You won't see any message that it is happening. If state locking fails,
 Terraform will not continue. You can disable state locking for most commands
```

Comment:
```suggestion
state. You do not see any message that it happens. If state locking fails,
Terraform does not continue. You can disable state locking for most commands
```

Suggest changing "won't" and "will not" to "do not" and "does not" to avoid using contractions and to avoid future tense. Suggest changing "is happening" to "happens" to use fewer words.